vscode-dotnet-sdk-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -54,6 +57,9 @@ suite('DotnetCoreAcquisitionExtension End to End', function()
   const mockDisplayWorker = new MockWindowDisplayWorker();
   const environmentVariableCollection = new MockEnvironmentVariableCollection();
   let extensionContext: vscode.ExtensionContext;
+  const mockExistingPaths: IExistingPaths = {
+    localExsitingPaths: [{extensionId: 'alternative.extension', path: 'foo'}]
```

Comment:
We should ad some more test data here, by adding a global path setting, and several mock extension ids, which:

1 does have its own local setting - make sure it takes precedence over the global one
and another which does not set the path, (not sure if this is correct and will work or not) and will either pick up the global one or fail. 

unfortunately there is no easy way to test without adding this weird test case behavior

Reply:
Oh whoops this was done, sorry! It's in the runtime extension and we dont need to test the sdk one so much ;)